src/com/google/javascript/jscomp/VariableReferenceCheck.java

```diff
@@ -227,11 +227,14 @@ private void checkVar(Var v, List<Reference> references) {
                 referenceNode, v);
         boolean letConstShadowsVar = v.getParentNode().isVar()
             && (reference.isLetDeclaration() || reference.isConstDeclaration());
```

Comment:
Since the changes to this file look like just refactoring, can we split them into a separate PR?